src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -113,6 +114,26 @@ function setAmazonPayConfig(adyenPaymentMethods) {
   }
 }
 
+function setInstallments(amount) {
```

Comment:
Can we return this information from the Backend if we add it to the sessions response?